workaround for:
  RuntimeError: The Poetry configuration is invalid:
    - Additional properties are not allowed ('group' was unexpected)
author: Kenneth Hoste (HPC-UGent)
--- pygam-0.9.0/pyproject.toml.orig	2024-01-05 22:13:47.399107878 +0100
+++ pygam-0.9.0/pyproject.toml	2024-01-05 22:13:52.323119792 +0100
@@ -12,19 +12,6 @@
 scipy = "^1.10.1"
 progressbar2 = "^4.2.0"
 
-[tool.poetry.group.dev.dependencies]
-pytest = "^7.2.2"
-flake8 = "^6.0.0"
-codecov = "^2.1.12"
-pytest-cov = "^4.0.0"
-mock = "^5.0.1"
-nbsphinx = "^0.9.0"
-sphinx-rtd-theme = "^1.2.0"
-sphinxcontrib-napoleon = "^0.7"
-ipython = "^8.11.0"
-pandas = "^1.5.3"
-black = "^23.1.0"
-
 [tool.black]
 line-length = 88
 skip-string-normalization = true
